sdk/kas_client.go

```diff
@@ -20,25 +22,21 @@ const (
 	secondsPerMinute = 60
 )
 
-type RequestBody struct {
-	KeyAccess       `json:"keyAccess"`
-	ClientPublicKey string `json:"clientPublicKey"`
-	Policy          string `json:"policy"`
-}
-
 type KASClient struct {
 	accessTokenSource auth.AccessTokenSource
 	dialOptions       []grpc.DialOption
 	sessionKey        *ocrypto.RsaKeyPair
 }
 
-// once the backend moves over we should use the same type that the golang backend uses here
-type rewrapRequestBody struct {
-	KeyAccess       KeyAccess `json:"keyAccess"`
-	Policy          string    `json:"policy,omitempty"`
-	Algorithm       string    `json:"algorithm,omitempty"`
-	ClientPublicKey string    `json:"clientPublicKey"`
-	SchemaVersion   string    `json:"schemaVersion,omitempty"`
+type KAOResult struct {
```

Comment:
Don't export this type as it is not exposed by any exported types or functions. 
```suggestion
type kaoResult struct {
```

Also, you are probably better off using a refactor option in your IDE than accepting this as it will just break stuff